app/assets/javascripts/models/beta/history.js

```diff
@@ -0,0 +1,22 @@
+import { sortBy } from 'underscore';
+
+export const getDate = (key, changes) => {
+  const [oldValue, newValue] = changes[key];
```

Comment:
Instead of passing changes and key, what do you think  about passing the old and new value directly? It'll make this function less coupled with the `changes` object.

Reply:
Sure! I liked of idea